workers/shared.py

```diff
@@ -86,20 +87,32 @@ def hdfs_to_local_path(hdfs_path):
 
     return filename
 
+def guess_ensembl_release(filepath):
+    try:
+        release = varcode.load_vcf(filepath)[0].ensembl.release
```

Comment:
@armish You can use `varcode.vcf.infer_genome_from_vcf` but then you're going to have to deal with the fact that not all VCFs are for humans :-)